data/mapcss-mapping.csv

```diff
@@ -747,7 +747,7 @@ amenity|parking_entrance|permissive;[amenity=parking_entrance][access=permissive
 barrier|cattle_grid;747;
 barrier|ditch;748;
 barrier|kissing_gate;749;
-deprecated|deprecated;750;x
+barrier|turnstile;750;
```

Comment:
Do we really need to distinguish stile and turnstile?

Reply:
imo it's worth it, stile and turnstile are very different things & I found it quite strange to see stiles at a train station 😄